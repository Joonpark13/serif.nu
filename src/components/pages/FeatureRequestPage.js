import React from 'react';
import { makeStyles } from '@material-ui/styles';
import { pageContainer, pageBody } from './common/styles';
import PageTitle from './common/PageTitle';

const useStyles = makeStyles({
  pageContainer,
  pageBody,
});

export default function FeatureRequestPage() {
  const classes = useStyles();
  return (
    <div className={classes.pageContainer}>
      <PageTitle title="Request a Feature" />
      <div className={classes.pageBody}>
        {(
          <iframe
            title="Request Feature Google Form"
            src="https://docs.google.com/forms/d/e/1FAIpQLScPNzvaEyny6E0RZpXiNJ1MTe-xN_XHas8siBnOEeQOmvmA-A/viewform?embedded=true"
            width="100%"
            height="1100"
            frameBorder="0"
            marginHeight="0"
            marginWidth="0"
          >
            Loading...
          </iframe>
        )}
      </div>
    </div>
  );
}
