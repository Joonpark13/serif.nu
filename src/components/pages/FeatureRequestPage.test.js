import React from 'react';
import { shallow } from 'enzyme';
import FeatureRequestPage from './FeatureRequestPage';

describe('FeatureRequestPage', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<FeatureRequestPage />);

    expect(wrapper.get(0)).toMatchSnapshot();
  });
});
