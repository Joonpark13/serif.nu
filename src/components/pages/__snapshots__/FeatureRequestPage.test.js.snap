// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeatureRequestPage should render correctly 1`] = `
<div
  className="makeStyles-pageContainer-1"
>
  <PageTitle
    title="Request a Feature"
  />
  <div
    className="makeStyles-pageBody-2"
  >
    <iframe
      frameBorder="0"
      height="1100"
      marginHeight="0"
      marginWidth="0"
      src="https://docs.google.com/forms/d/e/1FAIpQLScPNzvaEyny6E0RZpXiNJ1MTe-xN_XHas8siBnOEeQOmvmA-A/viewform?embedded=true"
      title="Request Feature Google Form"
      width="100%"
    >
      Loading...
    </iframe>
  </div>
</div>
`;
